rustls-platform-verifier/Cargo.toml

```diff
@@ -0,0 +1,76 @@
+[package]
```

Comment:
Could you get git to recognize this as a move/rename?

Reply:
I can give it a try, I'm not really sure why it didn't already. Maybe because the original filename is still there?

Reply:
The only way I was able to accomplish this was by entirely removing the original `Cargo.toml` in the first commit and then adding the workspace contents in a second. It does make this set of commits less atomic, but in exchange the history should be preserved. Thoughts?

Reply:
Probably not worth it.